spec/PluginClientSpec.php

```diff
@@ -45,4 +46,19 @@ function it_sends_async_request_with_underlying_client(HttpAsyncClient $asyncCli
         $this->beConstructedWith($asyncClient);
         $this->sendAsyncRequest($request)->shouldReturn($promise);
     }
+
+    function it_throws_loop_exception(HttpClient $client, RequestInterface $request)
+    {
+        $this->beConstructedWith($client, [new DefectuousPlugin()]);
+
+        $this->shouldThrow('Http\Client\Plugin\Exception\LoopException')->duringSendRequest($request);
+    }
+}
+
+class DefectuousPlugin implements Plugin
+{
+    public function handleRequest(RequestInterface $request, callable $next, callable $first)
```

Comment:
Try using the FQCN here. Might be a PHP 7, but. This workaround could temporarily solve it. Don't forget adding a warning, that we had to make our hands dirty because of that.